addons/web_editor/static/src/js/editor/snippets.editor.js

```diff
@@ -4087,7 +4087,7 @@ var SnippetsMenu = Widget.extend({
         // Use a setTimeout fallback to avoid locking the editor if the pointerup
         // is fired over an element which stops propagation for example.
         const enableTimeoutID = setTimeout(() => reenable(), 5000);
-        $(document).one('pointerup', () => {
+        $(document).add(this.$body).one('pointerup', () => {
```

Comment:
I'll move this commit into its own branch as QSM suggested but I'll quickly try to explain here:

It seems [this commit](https://github.com/odoo/odoo/commit/3ea7d6520c4f7595a5436f5045608d6451f78bbd) introduced a bug fix that when dragging an input from the side, and releasing over the editable, you would not change selection. I don't think it's super easily reproducible here, as the iframe would only pick up a `mouseup` event. Maybe it can be done in mass_mailing? 

I'll remove the change done in commit 1 too as they don't make much sense as it seems the bug happens when dragging from an input in a snippet option, and this would not be affected by the "pointerevent" bug.